dags/desktop_clients_daily.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+
+from airflow import models
+from airflow.contrib.operators.bigquery_operator import BigQueryOperator
+
+with models.DAG(
+    "clients_daily",
+    schedule_interval="0 1 * * *",
+    default_args={
+        "start_date": datetime.datetime(2019, 3, 1),
+        "email": ["dthorn@mozilla.com", "dataops+alerts@mozilla.com"],
+        "email_on_failure": True,
+        "email_on_retry": False,
+        "depends_on_past": True,
+        # If a task fails, retry it once after waiting at least 5 minutes
+        "retries": 0,
+        "retry_delay": datetime.timedelta(minutes=5),
+        "project_id": models.Variable.get("gcp_project"),
+    },
+) as dag:
+    clients_daily = BigQueryOperator(
+        task_id="clients_daily",
+        bql="sql/clients_daily_v7.sql",
+        destination_dataset_table="analysis.clients_daily_v7${{ds_nodash}}",
```

Comment:
I think we should template the destination `dataset_id`, whether that is something like this:
```suggestion
        destination_dataset_table="{{dataset_id}}.clients_daily_v7${{ds_nodash}}",
```
or more like this
```suggestion
        destination_dataset_table="clients_daily_v7${{ds_nodash}}",
        dataset_id="analysis",
```

Reply:
i will fix this along with the custom bq operator in a separate pr